Sources/TuistCache/Cache/URL+Cache.swift

```diff
@@ -3,27 +3,41 @@ import Foundation
 extension URL {
     static func apiCacheURL(hash: String,
```

Comment:
Considering these methods are only used by the factory, I think having a URL extension adding methods to URL that is scoped to one file is not necessary. I'd rather move these methods to the file where they are scoped to.

Reply:
Ok, I'll move them